.github/workflows/dependency-checker.yml

```diff
@@ -15,6 +15,9 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: 3.11
+        
+      - name: Install Dependencies
+        run: pip install packaging requests
```

Comment:
Please pin here or create a requirements file and `pip install -r` that. Sliding requirements just causes pain in trying to debug build failures.